tf2onnx/tfonnx.py

```diff
@@ -1536,12 +1547,13 @@ def mknode(op_type, inputs, opname, **kwargs):
     "Relu6": (relu6_op, []),
     "Reshape": (reshape_op, ["Reshape"]),
     "Rsqrt": (rsqrt_op, []),
+    "Select": (select_op, []),
```

Comment:
dpesn't this want to go into the opset8 table?

Reply:
nice catch! will fix it .